global.json

```diff
@@ -1,5 +1,5 @@
 {
   "sdk": {
-    "version": "6.0.101"
+    "version": "6.0.X"
```

Comment:
https://learn.microsoft.com/en-us/dotnet/core/tools/global-json

"Doesn't have wildcard support; that is, you must specify the full version number."

You need to configure the rollForward like this:

 "rollForward": "latestFeature"